mapper/orm/src/main/java/org/hibernate/search/mapper/orm/massindexing/MassIndexingFailureContext.java

```diff
@@ -80,6 +82,24 @@ public Object getFailingOperation() {
 		return failingOperation();
 	}
 
+	@Override
+	public boolean equals(Object o) {
```

Comment:
Uh no, I don't think it's a good idea to add this just for tests... I don't quite see a use case. Plus you forgot to override equals/hashcode in subclasses.

No, really, let's not do this. If you want to check that `failingOperation` and `throwable` are the same in your test, just do that explicitly? `assertThat( context1.failingOperation() ).isEqualTo( ...)`?